client/my-sites/people/edit-team-member-form/with-update-user.js

```diff
@@ -0,0 +1,88 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { useDispatch } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import useUpdateUserMutation from 'calypso/data/users/use-update-user-mutation';
+import { errorNotice, successNotice, plainNotice } from 'calypso/state/notices/actions';
+import { useTranslate } from 'i18n-calypso';
+
+const useSuccessNotice = ( isSuccess, user ) => {
+	const dispatch = useDispatch();
+	const translate = useTranslate();
+
+	React.useEffect( () => {
+		isSuccess &&
+			dispatch(
+				successNotice(
+					translate( 'Successfully updated @%(user)s', {
+						args: { user: user?.login },
+						context: 'Success message after a user has been modified.',
+					} ),
+					{ id: 'update-user-notice' }
+				)
+			);
+		// Note: We only want to run this effect in case `isSuccess`
+		// changes and ignore changes to other deps like `user`.
+		// eslint-disable-next-line react-hooks/exhaustive-deps
+	}, [ isSuccess ] );
+};
+
+const useErrorNotice = ( isError, user ) => {
+	const dispatch = useDispatch();
+	const translate = useTranslate();
+
+	React.useEffect( () => {
+		isError &&
+			dispatch(
+				errorNotice(
+					translate( 'There was an error updating @%(user)s', {
+						args: { user: user?.login },
+						context: 'Error message after A site has failed to perform actions on a user.',
+					} ),
+					{ id: 'update-user-notice' }
+				)
+			);
+		// Note: We only want to run this effect in case `isError`
+		// changes and ignore changes to other deps like `user`.
+		// eslint-disable-next-line react-hooks/exhaustive-deps
+	}, [ isError ] );
+};
+
+const usePendingNotice = ( isPending, user ) => {
```

Comment:
I've seen those "in progress" notices in plugins too, and they feel weird to me. I believe that the UI should be pretty straightforward that we're doing an update - UI controls are deactivated while submitting, etc, so that should be enough - like in Site Settings. I guess we can consult a designer if we're not sure, but my inclination would be to remove them 😉

Reply:
I removed `usePendingNotice` in https://github.com/Automattic/wp-calypso/pull/50992/commits/5119d9e337413370129b1c4183ccea7508660f71